### ISSUE TITLE
Can I change log Severity at runtime by the linux command kill ?

### ISSUE THREAD

**Issue by AlexYIN** [SergiusTheBest/plog#104]
Can I change log Severity by the linux command kill at runtime?

Such as, the current log Severity is INFO, but I want to trace more information, so I need to change the log Severity to DEBUG with the kill command and without the program restart .

**Comment by SergiusTheBest** [comment]
Yes, you can but you need to write and register a signal handler (that will react on kill command). 
```cpp
void handle_signal(int signal)
{
    plog::get()->setMaxSeverity(plog::verbose);
}
```

**Comment by elfring** [comment]
I suggest to take another look at [asynchronous-signal safety](https://stackoverflow.com/questions/34467694/why-only-async-signal-safe-functions-can-be-called-from-signal-handlers-safely "Why only async-signal-safe functions can be called from signal handlers safely?") for calls of member functions.
